technische-universitat-dresden-finanzwirtschaft-und-finanzdienstleistungen-note.csl

```diff
@@ -0,0 +1,209 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" demote-non-dropping-particle="sort-only" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Technische Universität Dresden - Finanzwirtschaft und Finanzdienstleistungen (note)</title>
+    <title-short>TUD Betriebswirtschaftslehre (note)</title-short>
+    <id>http://www.zotero.org/styles/technische-universitat-dresden-finanzwirtschaft-und-finanzdienstleistungen-note</id>
+    <link href="http://www.zotero.org/styles/technische-universitat-dresden-finanzwirtschaft-und-finanzdienstleistungen-note" rel="self"/>
+    <author>
+      <name>Jens Mittelbach</name>
+      <email>jens.mittelbach@slub-dresden.de</email>
+      <uri>http://www.slub-dresden.de</uri>
+    </author>
+    <contributor>
+      <name>Julian Onions</name>
+      <email>julian.onions@gmail.com</email>
+    </contributor>
+    <category citation-format="note"/>
+    <category field="generic-base"/>
```

Comment:
I think we normally use `social_science` also for economics here, at least that would be consistent with with your other style https://www.zotero.org/styles/technische-universitat-dresden-wirtschaftswissenschaften?source=1 . I suppose that these two chairs use different citaiton styles also they are at the same departement, correct?